src/annotator/anchoring/pdf.js

```diff
@@ -436,21 +459,17 @@ export function anchor(root, selectors) {
   /** @type {Promise<Range>} */
   let result = Promise.reject('unable to anchor');
 
-  const checkQuote = range => {
-    if (quote && quote.exact !== range.toString()) {
-      throw new Error('quote mismatch');
-    }
-    return range;
-  };
-
   if (position) {
     result = result.catch(() => {
       return findPage(position.start).then(({ index, offset, textContent }) => {
         const start = position.start - offset;
         const end = position.end - offset;
         const length = end - start;
 
-        checkQuote(textContent.substr(start, length));
+        const matchedText = textContent.substr(start, length);
+        if (quote && quote.exact !== matchedText) {
+          throw new Error('quote mismatch');
+        }
```

Comment:
There is no change to the logic here, I just inlined `checkQuote` to remove a misleading `range` parameter which is, in fact, a `string` rather than a `Range`.